API/Reference/Project-Lockdown.v1.json

```diff
@@ -9906,7 +9908,7 @@
         }
       ]
     },
-    "/Territories/{PLDCode}/{Depth}": {
+    "/Territories/{PLDCode}/{Depth}/{Format}": {
```

Comment:
as discussed, we should probably change to using query parameters, ie `/terratories/plcode?depth=1&format=json`
but that can be managed in another separate PR